octobot_trading/channels/positions.py

```diff
@@ -27,6 +27,8 @@ async def push(self, positions, is_closed=False, is_liquidated=False, is_from_bo
     async def perform(self, positions, is_closed=False, is_liquidated=False, is_from_bot=True):
         try:
             for position in positions:
+                if not position:
```

Comment:
a position can be None ?

Reply:
Yes, on binance future, when there is open orders but no position, binance is giving `[None]` instead of `[]` when requesting open positions.